homeassistant/components/energyzero/coordinator.py

```diff
@@ -43,7 +50,13 @@ def __init__(self, hass: HomeAssistant) -> None:
             update_interval=SCAN_INTERVAL,
         )
 
-        self.energyzero = EnergyZero(session=async_get_clientsession(hass))
+        self.gas_modifier = Template(gas_modifier, hass)
+        self.energy_modifier = Template(energy_modifier, hass)
+
+        self.energyzero = EnergyZero(
+            session=async_get_clientsession(hass),
+            incl_btw="false",
```

Comment:
Careful here. The string `"false"` is considered a truthy value and would pass `if incl_btw:`

Reply:
It is typed a string and a "feature" of https://pypi.org/project/energyzero/